cogeo_mosaic/supermorecado.py

```diff
@@ -145,9 +151,11 @@ def make_transform(self, tilerange: Dict, zoom: int) -> Affine:
 
         return Affine(xcell, 0, ulx, 0, -ycell, uly)
 
-    def burn(self, polys: Sequence[Dict[Any, Any]], zoom: int) -> numpy.ndarray:
+    def burn(
+        self, polys: Sequence[Dict[Any, Any]], zoom: int, truncate: bool = True
+    ) -> numpy.ndarray:
         """Burn geometries to Tiles."""
-        bounds = find_extrema(polys)
+        bounds = self.find_extrema(polys, truncate=truncate)
```

Comment:
I have added a `truncate` options because for sometimes you could want to create tiles outside the tms bbox